tests/test-org-ql.el

```diff
@@ -2111,7 +2111,84 @@ with keyword arg NOW in PLIST."
           (it "Refuses to link to non-file-backed buffer"
             (expect (var-after-link-save-open 'org-ql-view-buffers-files link-buffer query
                                               :buffer link-buffer)
-                    :to-throw 'user-error '("Views that search non-file-backed buffers can’t be linked to"))))))
+                    :to-throw 'user-error '("Views that search non-file-backed buffers can’t be linked to"))))
+        (describe "Completion for Files/buffers"
+          (describe "Contracting org-ql-view-buffers-files"
+            (it "org-agenda-files from list"
+              (spy-on 'org-agenda-files :and-return-value temp-filenames)
+              (expect (org-ql-view--contract-buffers-files temp-filenames) :to-equal "org-agenda-files"))
+            (it "org-directory"
+              (spy-on 'org-ql-search-directories-files :and-return-value temp-filenames)
+              (spy-on 'org-agenda-files :and-return-value '())
```

Comment:
Nitpick: since this is tested first in the function in question, it would be more, shall we say, symmetrical, to put it first in this test.  :)

Reply:
Re-ordering :D